### PR TITLE
docs/Update wallet section

### LINKED ISSUE #1245: User-friendly wallets section
We currently have this page: https://github.com/filecoin-project/filecoin-docs/blob/main/content/en/about-filecoin/managing-assets/index.md

Problems:
- It assumes you're an expert user who has an automatically-created wallet via a lotus node

It should:
- Briefly describe what wallets are, for the completely uninitiated (eg IPFS user who won some FIL tokens in a hackathon)
- Explain the difference between different types of wallets (mobile, desktop, hardware, CLI-wallets-automatically-created-with-lotus-nodes, etc.)
- Remove all assumptions that you're using a wallet created via lotus node
- Consider a new, more straightforward title for this page

Other improvement ideas are welcome.

### PR DESCRIPTION
**Pull request by dawnkelly09**
A start on the updated wallet section. Fixes #1245 

There are some pretty significant possible changes so I thought maybe we could hash out some questions I have on this Google doc? It should open for anyone within PL and let you edit or comment. 

https://docs.google.com/document/d/19H1ptbObGtNYaGrtirygnWaPsGItw9BOUMf6iUBvYOI/edit?usp=sharing

@mishmosh

### CODE REVIEW

File: content/en/about-filecoin/managing-assets/index.md [hunk 1]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
+
+With non-custodial wallets, the user holds their own private key and has total control over the movement of assets. Non-custodial wallet users are also solely responsible for securing their assets. Protecting private keys and seed phrases is extremely important to prevent fraudulent access to assets. Most non-custodial wallets do not have KYC requirements and allow users to maintain anonymity.
+
+### Hot vs cold
+
+Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:23]:
```suggestion
Hot wallet refers to any wallet which is permanently connected to the internet. They can be mobile, desktop, or browser based. Hot wallets make it faster and easier to access digital access, but being could be vulnerable to online attacks. Therefore, it is recommended to keep large balances in cold wallets and only use hot wallets to hold funds you need to access frequently.
```
```suggestion
Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 2]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
+
+### Custodial vs non-custodial
+
+A custodial wallet is one which is hosted by an exchange. These wallets can come with a greater sense of security as the exchange holds the private keys. Many will also provide insurance to help recover funds lost to fraud, similar to centralized stock or bank accounts. The tradeoff is assets in custodial wallets can be frozen or seized by the associated exchange. Most custodial wallets also have KYC (Know Your Customer) requirements which require users to verify their identity in order to use the service.
+
+With non-custodial wallets, the user holds their own private key and has total control over the movement of assets. Non-custodial wallet users are also solely responsible for securing their assets. Protecting private keys and seed phrases is extremely important to prevent fraudulent access to assets. Most non-custodial wallets do not have KYC requirements and allow users to maintain anonymity.
+
+### Hot vs cold
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:21]:
```suggestion
### Hot vs. cold
```
```suggestion
### Hot vs cold
```

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:21]:
```suggestion
### Hot vs. cold
```
```suggestion
### Hot vs cold
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 3]
```diff
@@ -9,13 +9,31 @@ weight: 60
 
 ## Wallets
 
+<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
+Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
+
+### Custodial vs non-custodial
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:15]:
```suggestion
### Custodial vs. non-custodial
```
```suggestion
### Custodial vs non-custodial
```

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:15]:
```suggestion
### Custodial vs. non-custodial
```
```suggestion
### Custodial vs non-custodial
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Managing assets"
-description: "Many interactions with the Filecoin network require FIL to process. This page explains the different denominations of FIL, how to store it, and how to aquire it."
+title: "FIL token"
+description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:3]:
```suggestion
description: "FIL is the native token of the Filecoin network. This page explains how to acquire, store, and use FIL tokens."
```
```suggestion
description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Simplify a lot. Also, while hackathon payments is one use-case that's not well-served by the current docs, it should not be called out at the network documentation level.

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:3]:
```suggestion
description: "FIL is the native token of the Filecoin network. This page explains how to acquire, store, and use FIL tokens."
```
```suggestion
description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Simplify a lot. Also, while hackathon payments is one use-case that's not well-served by the current docs, it should not be called out at the network documentation level.

File: content/en/about-filecoin/managing-assets/index.md [hunk 5]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
+
+With non-custodial wallets, the user holds their own private key and has total control over the movement of assets. Non-custodial wallet users are also solely responsible for securing their assets. Protecting private keys and seed phrases is extremely important to prevent fraudulent access to assets. Most non-custodial wallets do not have KYC requirements and allow users to maintain anonymity.
+
+### Hot vs cold
+
+Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.
+
+Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely.
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:25]:
```suggestion
Cold wallets most commonly refer to hardware wallet devices shaped like a USB stick. They are typically offline, and only connected to the internet for transactions. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key, which makes them more resistant to loss via theft. Cold wallets can be less convenient, and are most useful for storing larger balances securely. 
```
```suggestion
Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 
```

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:25]:
```suggestion
Cold wallets most commonly refer to hardware wallet devices shaped like a USB stick. They are typically offline, and only connected to the internet for transactions. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key, which makes them more resistant to loss via theft. Cold wallets can be less convenient, and are most useful for storing larger balances securely. 
```
```suggestion
Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 6]
```diff
@@ -9,13 +9,31 @@ weight: 60
 
 ## Wallets
 
+<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
+Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:13]:
```suggestion
`FIL` is the native token of the Filecoin network. FIL tokens are used as an economic incentive to ensure data is stored reliably over time.

Wallets provide a way to securely store digital assets. Here are some wallets with FIL support that you might consider.

Note: If you are already running your own lotus node, you can also [manage FIL wallets from the command line](https://lotus.filecoin.io/lotus/manage/manage-fil/).

```
```suggestion
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Let's leave the wrapped FIL reference out, people can find it in the sidebar if desired.

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:13]:
```suggestion
`FIL` is the native token of the Filecoin network. FIL tokens are used as an economic incentive to ensure data is stored reliably over time.

```
```suggestion
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Let's leave the wrapped FIL reference out, people can find it in the sidebar if desired.

File: content/en/about-filecoin/managing-assets/index.md [hunk 7]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
 Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
 
-{{< alert icon="warning" >}}
-**Unaudited Wallets**
+We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
 
-We do not support any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet. Wallets that _have_ gone through an audit have had their codebase checked by a recognized security firm for security vulnerabilities and potential leaks. However, just because a wallet has had an audit does not mean that it's 100% bug-proof. Be incredibly cautious when using unaudited wallets.
-{{< /alert >}}
 
  Name                                           | Description                                                                                                                          | Audited |
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:38]: Let's move the wallets list above the tradeoffs (hot vs cold, security, etc.)

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:38]: Let's move the wallets list above the tradeoffs (hot vs cold, security, etc.)

File: content/en/about-filecoin/managing-assets/index.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Managing assets"
-description: "Many interactions with the Filecoin network require FIL to process. This page explains the different denominations of FIL, how to store it, and how to aquire it."
```
Comment (pap1ll0n) [inline content/en/about-filecoin/managing-assets/index.md:3]: acquire

File: content/en/about-filecoin/managing-assets/index.md (hunk 3 above)
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:15]: Now that I see the whole article, I think we can remove this section on custodial vs non-custodial. It's extremely well written! But less immediately relevant for novice users, and more expert users can find this information elsewhere.

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:15]: Now that I see the whole article, I think we can remove this section on custodial vs non-custodial. It's extremely well written! But less immediately relevant for novice users, and more expert users can find this information elsewhere.

File: content/en/about-filecoin/managing-assets/index.md [hunk 9]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
+
+### Security
+
+Wallets that have gone through an audit have had their codebase checked by a recognized security firm for security vulnerabilities and potential leaks. However, just because a wallet has had an audit does not mean that it’s 100% bug-proof. Be incredibly cautious when using unaudited wallets.
+
+Never share your seed phrase, password, or private keys. Bad actors will often use social engineering tactics such as phishing emails or posing as customer service or tech support to lure users into handing over their private key or seed phrase.
+
 Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:33]:
```suggestion
```
```suggestion
Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
```

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:33]:
```suggestion
```
```suggestion
Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 10]
```diff
@@ -9,13 +9,31 @@ weight: 60
...
 
-{{< alert icon="warning" >}}
-**Unaudited Wallets**
+We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
```
Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:35]:
```suggestion
We do not provide support for any of these wallets.
```
```suggestion
We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
```

This isn't a great recommendation for first-time token users. Let's remove any recommendation at all, and make sure the list & rest of article are helpful.

Comment (mishmosh) [inline content/en/about-filecoin/managing-assets/index.md:35]:
```suggestion
We do not provide support for any of these wallets.
```
```suggestion
We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
```

This isn't a great recommendation for first-time token users. Let's remove any recommendation at all, and make sure the list & rest of article are helpful.

File: content/en/about-filecoin/managing-assets/index.md (hunk 7 above)
Comment (mishmosh) [reply content/en/about-filecoin/managing-assets/index.md:38]: Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

Comment (mishmosh) [reply content/en/about-filecoin/managing-assets/index.md:38]: Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

**Review by mishmosh — COMMENTED** [review]
Great start! Some nitpicky comments & suggestions due to security, positioning, etc. Once you've addressed my comments and incorporated @pap1ll0n feedback, please assign `johnnymatthews` for review.

File: content/en/about-filecoin/managing-assets/index.md (hunk 3 above)
Comment (dawnkelly09) [reply content/en/about-filecoin/managing-assets/index.md]: Section removed.

File: content/en/about-filecoin/managing-assets/index.md (hunk 7 above)
Comment (dawnkelly09) [reply content/en/about-filecoin/managing-assets/index.md:20]: Moved list up in doc

**Comment by dawnkelly09** [comment]
Ok, @johnnymatthews this is ready for your review. Thank you!

File: content/en/about-filecoin/managing-assets/index.md [hunk 11]
```diff
@@ -12,7 +12,7 @@ weight: 60
 <!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
 Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
 
-### Custodial vs non-custodial
+### Custodial vs. non-custodial
```
Comment (johnnymatthews) [inline content/en/about-filecoin/managing-assets/index.md]:
```suggestion
### Custodial versus non-custodial
```

File: content/en/about-filecoin/managing-assets/index.md [hunk 12]
```diff
@@ -28,6 +28,18 @@ We do not support any of these wallets. We recommend that you use an audited wal
 | [MathWallet](https://mathwallet.org/en-us/)    | A multi-currency wallet.                                                                                                             | Unknown |
 | [Trust wallet](https://trustwallet.com/)                                   | A multi-currency wallet. This is the official wallet of Binance.                                                                     | Unknown |
 
+### Hot vs. cold
```
Comment (johnnymatthews) [inline content/en/about-filecoin/managing-assets/index.md:31]:
```suggestion
### Hot versus cold
```

**Review by johnnymatthews — APPROVED** [review]

**Pull request merged**